### ISSUE TITLE
Setting RxJavaPlugins error handler may lead to unexpected behaviour

### ISSUE THREAD

**Issue by Dreiko** [onfido/onfido-android-sdk#124]
#### What was the expected behaviour?

SDK should not use global RxJavaPlugins

#### What happened instead?

`com.onfido.android.sdk.capture.ui.camera.CapturePresenter` has `handleUndeliverableException` method which calls `RxJavaPlugins.setErrorHandler()`.

It makes impossible to use RxJavaPlugins.errorHandler in caller code.

Workaround
```
# This is a configuration file for ProGuard.

-assumenosideeffects class com.onfido.android.sdk.capture.ui.camera.CapturePresenter {
    private void handleUndeliverableException();
}
```

#### Version info:

- `onfido-capture-sdk` version: 6.0 and below

#### Additional LogCat output:
```
    Caused by: java.lang.IllegalStateException: Plugins can't be changed anymore
        at io.reactivex.plugins.RxJavaPlugins.setErrorHandler(RxJavaPlugins.java:550)
        at com.onfido.android.sdk.capture.ui.camera.CapturePresenter.handleUndeliverableException(Unknown Source:7)
        at com.onfido.android.sdk.capture.ui.camera.CapturePresenter.<init>(Unknown Source:94)
        at com.onfido.android.sdk.capture.ui.camera.CapturePresenter.<init>(Unknown Source:45)
        at com.onfido.android.sdk.capture.common.di.SdkModule.provideCapturePresenter(Unknown Source:92)
        at com.onfido.android.sdk.capture.common.di.SdkModule_ProvideCapturePresenterFactory.get(Unknown Source:98)
        at com.onfido.android.sdk.capture.common.di.SdkModule_ProvideCapturePresenterFactory.get(Unknown Source:0)
        at com.onfido.android.sdk.capture.ui.camera.CaptureActivity_MembersInjector.injectMembers(Unknown Source:14)
        at com.onfido.android.sdk.capture.ui.camera.CaptureActivity_MembersInjector.injectMembers(Unknown Source:2)
        at com.onfido.android.sdk.capture.common.di.DaggerSdkComponent.inject(Unknown Source:2)
        at com.onfido.android.sdk.capture.ui.camera.CaptureActivity.setupPresenter(Unknown Source:12)
        at com.onfido.android.sdk.capture.ui.camera.CaptureActivity.onCreate(Unknown Source:112)
        at android.app.Activity.performCreate(Activity.java:7802)
        at android.app.Activity.performCreate(Activity.java:7791)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1307)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3246)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3410) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:83) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2017) 
        at android.os.Handler.dispatchMessage(Handler.java:107) 
        at android.os.Looper.loop(Looper.java:214) 
        at android.app.ActivityThread.main(ActivityThread.java:7403) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:935)
```

**Comment by burakeregar** [comment]
Hi @Dreiko,

Thanks for reporting the issue.

This is in our backlog now, we will update here once it's fixed.

**Comment by burakeregar** [comment]
This issue has been fixed as part of version [7.1.0](https://github.com/onfido/onfido-android-sdk/releases/tag/7.1.0)
